ghost/core/core/server/data/migrations/versions/5.19/2022-09-02-20-52-backfill-new-product-columns.js

```diff
@@ -0,0 +1,37 @@
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const rows = await knex('products as t') // eslint-disable-line no-restricted-syntax
+            .select(
+                't.id as id',
+                'mp.amount as monthly_price',
+                'yp.amount as yearly_price',
+                knex.raw('coalesce(yp.currency, mp.currency) as currency')
+            )
+            .leftJoin('stripe_prices AS mp', 't.monthly_price_id', 'mp.id')
+            .leftJoin('stripe_prices AS yp', 't.yearly_price_id', 'yp.id')
+            .where('t.type', 'paid');
+
+        if (!rows) {
```

Comment:
Do we need a `.length` here? Otherwise `rows` is `[]` which is still truthy

Reply:
Good catch